.github/workflows/update-fixed-env.yaml

```diff
@@ -1,39 +1,34 @@
-name: Fixed Environment YAML Monitor
+name: Environment Checker
 
 on:
   push:
     branches:
     - master
     paths:
-    - 'env/environment.yaml'
+    - 'envs/environment.yaml'
 
 jobs:
-  update_environment_fixed:
+  update-environment-fixed:
+    name: Update environment.fixed.yaml
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout Repository
-      uses: actions/checkout@v4
+    - uses: actions/checkout@v4
 
-    - name: Setup micromamba
-      uses: mamba-org/setup-micromamba@v1
+    - name: Setup conda
+      uses: conda-incubator/setup-miniconda@v3
       with:
-        micromamba-version: latest
+        activate-environment: pypsa-eur
         environment-file: envs/environment.yaml
-        log-level: debug
-        init-shell: bash
-        cache-environment: true
-        cache-downloads: true
 
     - name: Update environment.fixed.yaml
       run: |
-        mamba env export --file envs/environment.fixed.yaml  --no-builds
+        conda env export --name pypsa-eur > envs/environment.fixed.yaml
```

Comment:
Wouldn't it be better with `--no-builds`?

Reply:
Thanks!